diff --git a/cmake/packages/litert/litert.cmake b/cmake/packages/litert/litert.cmake
index f314c929..bfa13dbc 100644
--- a/cmake/packages/litert/litert.cmake
+++ b/cmake/packages/litert/litert.cmake
@@ -141,6 +141,9 @@ ExternalProject_Add(
     -DLITERT_DISABLE_KLEIDIAI=ON
     -DLITERT_BUILD_C_API=ON
     -DLITERT_BUILD_TOOLS=OFF
+    -DANDROID_EGL_LIB=/system/lib64/libEGL.so
+    -DANDROID_GLESV2_LIB=/system/lib64/libGLESv2.so
+    -DANDROID_GLESV3_LIB=/system/lib64/libGLESv3.so
     "-DCMAKE_EXE_LINKER_FLAGS=-L${ABSL_LIB_DIR} -L${PROTO_LIB_DIR} -L${FLATBUFFERS_LIB_DIR} -L${TFLITE_LIB_DIR}"
 
     # TODO(totero): Organize these flags.
